connexion/decorators/response.py

```diff
@@ -65,11 +65,15 @@ def validate_response(self, data, status_code, headers):
                 raise NonConformingResponseBody(message=str(e))
 
         if response_definition and response_definition.get("headers"):
-            response_definition_header_keys = response_definition.get("headers").keys()
-            if not all(item in headers.keys() for item in response_definition_header_keys):
-                raise NonConformingResponseHeaders(
-                    message="Keys in header don't match response specification. Difference: %s"
-                    % list(set(headers.keys()).symmetric_difference(set(response_definition_header_keys))))
+            # converting to set is needed to support python 2.7
```

Comment:
Do we need this comment? If it is removed the tests are going to break, plus your commit message is clear to say why it was added.